app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java

```diff
@@ -158,7 +160,8 @@ public void setRetryTimer() {
     }
 
     public void setFailoverTimer() {
-        if (CollectionServiceStarter.isBTWixel(getApplicationContext())|| CollectionServiceStarter.isDexbridgeWixel(getApplicationContext())) {
+        if (CollectionServiceStarter.isBTWixel(getApplicationContext())|| CollectionServiceStarter.isDexbridgeWixel(getApplicationContext())
```

Comment:
we may want to think about pulling these out into a boolean in the collection service starter that goes something like 
`CollectionServiceStarter.shouldRunDexCollectionService(context)`
Not something needed for this pr, but something that would probably be good to consider